man/machine-id.xml

```diff
@@ -104,6 +104,8 @@
     (on KVM systems), the Xen hypervisor <filename>uuid</filename>, and finally a randomly
     generated UUID.</para>
 
+    <para>The vm logic can be enabled manually by setting the <varname>systemd.machine_id_smbios</varname> kernel command line option.</para>
```

Comment:
Don't call this the "vm logic". It's not really a "VM logic", it's a logic of initializing the machine ID from the firmware uuid, that just happens to be done in the VM case only so far.